src/pool-bin/libraries/BinHelper.sol

```diff
@@ -169,16 +169,16 @@ library BinHelper {
     /// @param amountsIn The amounts of tokens to add
     /// @param totalSupply The total supply of the liquidity book
     /// @param shares The share of the liquidity book that the user will receive
-    /// @return fees The encoded fees that will be charged (including protocol and LP fee)
-    //// @return feeForProtocol The encoded protocol fee that will be charged
+    /// @return feesAmount The encoded fees that will be charged (including protocol and LP fee)
+    /// @return feeAmountToProtocol The encoded protocol fee that will be charged
     function getCompositionFees(
```

Comment:
(optional) 😆 i might even rename this `getCompositionFees -> getCompositionFeesAmount`